All queries failed when one BE network or disk has issue

Steps to reproduce the behavior:
1. Use iptables drop BE brpc port
2. query


```
2020-09-19 17:40:57,418 WARN 429 [BackendServiceProxy.execPlanFragmentAsync():116] Execute plan fragment catch a exception, address=172.26
.34.141:9990
java.lang.RuntimeException: Unable to validate object
at com.baidu.jprotobuf.pbrpc.transport.ChannelPool.getChannel(ChannelPool.java:85) ~[jprotobuf-rpc-core-3.5.21.jar:?]
at com.baidu.jprotobuf.pbrpc.transport.RpcChannel.getConnection(RpcChannel.java:72) ~[jprotobuf-rpc-core-3.5.21.jar:?]
at com.baidu.jprotobuf.pbrpc.client.ProtobufRpcProxy.invoke(ProtobufRpcProxy.java:486) ~[jprotobuf-rpc-core-3.5.21.jar:?]
at com.sun.proxy.$Proxy35.execPlanFragmentAsync(Unknown Source) ~[?:?]
at org.apache.doris.rpc.BackendServiceProxy.execPlanFragmentAsync(BackendServiceProxy.java:99) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.Coordinator$BackendExecState.execRemoteFragmentAsync(Coordinator.java:1471) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.Coordinator.exec(Coordinator.java:488) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:608) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:254) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:209) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:343) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:528) ~[dorisdb-fe.jar:?]
at org.apache.doris.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:50) ~[dorisdb-fe.jar:?]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_252]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_252]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: java.util.NoSuchElementException: Unable to validate object
at org.apache.commons.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:502) ~[commons-pool2-2.2.jar:2.2]
at org.apache.commons.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:361) ~[commons-pool2-2.2.jar:2.2]
at com.baidu.jprotobuf.pbrpc.transport.ChannelPool.getChannel(ChannelPool.java:79) ~[jprotobuf-rpc-core-3.5.21.jar:?]
... 15 more
```

```
2020-09-19 17:40:57,418 INFO 429 [QeProcessorImpl.unregisterQuery():78] deregister query id 9c01fd8356c146a9-8e6a454bcf396e02
2020-09-19 17:40:57,419 WARN 429 [StmtExecutor.execute():318] execute Exception
java.lang.NullPointerException: null
at org.apache.doris.qe.Coordinator$BackendExecState$1.get(Coordinator.java:1495) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.Coordinator$BackendExecState$1.get(Coordinator.java:1505) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.Coordinator$BackendExecState$1.get(Coordinator.java:1475) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.Coordinator.exec(Coordinator.java:496) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.StmtExecutor.handleQueryStmt(StmtExecutor.java:608) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:254) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:209) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:343) ~[dorisdb-fe.jar:?]
at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:528) ~[dorisdb-fe.jar:?]
at org.apache.doris.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:50) ~[dorisdb-fe.jar:?]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_252]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_252]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
```

--- a/fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java
+++ b/fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java
@@ -1472,7 +1472,7 @@ public class Coordinator {
             } catch (RpcException e) {
                 // DO NOT throw exception here, return a complete future with error code,
                 // so that the following logic will cancel the fragment.
-                Future<PExecPlanFragmentResult> future = new Future<PExecPlanFragmentResult>() {
+                return new Future<PExecPlanFragmentResult>() {
                     @Override
                     public boolean cancel(boolean mayInterruptIfRunning) {
                         return false;
@@ -1489,9 +1489,10 @@ public class Coordinator {
                     }
 
                     @Override
-                    public PExecPlanFragmentResult get() throws InterruptedException, ExecutionException {
+                    public PExecPlanFragmentResult get() {
                         PExecPlanFragmentResult result = new PExecPlanFragmentResult();
                         PStatus pStatus = new PStatus();
+                        pStatus.error_msgs = Lists.newArrayList();
                         pStatus.error_msgs.add(e.getMessage());
                         // use THRIFT_RPC_ERROR so that this BE will be added to the blacklist later.
                         pStatus.status_code = TStatusCode.THRIFT_RPC_ERROR.getValue();
@@ -1500,12 +1501,10 @@ public class Coordinator {
                     }
 
                     @Override
-                    public PExecPlanFragmentResult get(long timeout, TimeUnit unit)
-                            throws InterruptedException, ExecutionException, TimeoutException {
+                    public PExecPlanFragmentResult get(long timeout, TimeUnit unit) {
                         return get();
                     }
                 };
-                return future;
             }
         }
 

--- a/fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java
+++ b/fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java
@@ -254,7 +254,7 @@ public class StmtExecutor {
                             throw e;
                         }
                         if (!context.getMysqlChannel().isSend()) {
-                            LOG.warn("retry {} times. stmt: {}", (i + 1), context.getStmtId());
+                            LOG.warn("retry {} times. stmt: {}", (i + 1), parsedStmt.getOrigStmt().originStmt);
                         } else {
                             throw e;
                         }

